client/my-sites/themes/themes-magic-search-card/index.jsx

```diff
@@ -296,7 +296,7 @@ class ThemesMagicSearchCard extends React.Component {
 							</div>
 						}
 						{
-							( isPremiumThemesEnabled && ( config.isEnabled( 'jetpack/pijp' ) || ! isJetpack ) ) &&
+							isPremiumThemesEnabled &&
```

Comment:
Hmm, testing this in `development` mode, there's no `<SegementedControl />` :slightly_frowning_face: